src/components/Profile.js

```diff
@@ -41,21 +41,30 @@ export default function Profile() {
   return (
     <DrawerLayout>
       <Grid>
-        { myProfile !== null ? <ProfileInfo myProfile={myProfile} /> : <>No Data</>}
+        {myProfile !== null ? (
+          <ProfileInfo myProfile={myProfile} />
+        ) : (
+          <>No Data</>
+        )}
       </Grid>
       <Grid container style={{ marginBottom: '21px' }}>
-        <Grid item sm={4} style={{ marginRight: '21px' }}>
+        <Grid
+          item
+          xs={12}
+          sm={5}
+          style={{ marginRight: '21px', marginBottom: '21px' }}
+        >
           <CommitChart />
         </Grid>
-        <Grid item sm>
+        <Grid item xs={12} sm>
```

Comment:
sm has no values set?
if you want to set some values for sm,  do it. Otherwise, remove the sm prop